fish_prompt.fish

```diff
@@ -895,16 +896,30 @@ function __bobthefish_prompt_desk -S -d 'Display current desk environment'
     set_color normal
 end
 
-function __bobthefish_prompt_nvm -S -d 'Display current node version through NVM'
-    [ "$theme_display_nvm" = 'yes' -a -n "$NVM_DIR" ]
+function __bobthefish_prompt_node -S -d 'Display current node version'
+    [ "$theme_display_node" = "yes" -o "$theme_display_nvm" = 'yes' ]
     or return
 
-    set -l node_version (nvm current 2> /dev/null)
+    set -l node_manager
+    set -l node_manager_dir
+
+    if type -fq nvm
+      set node_manager "nvm"
+      set node_manager_dir $NVM_DIR
+    else if type -fq fnm
+      set node_manager "fnm"
+      set node_manager_dir $FNM_DIR
+    end
+
+    [ -n "$node_manager_dir" ]
+    or return
+
+    set -l node_version ("$node_manager" current)
 
     [ -z $node_version -o "$node_version" = 'none' -o "$node_version" = 'system' ]
     and return
 
-    __bobthefish_start_segment $color_nvm
+    __bobthefish_start_segment $color_node
```

Comment:
i know you've updated the default color schemes, but bobthefish also allows custom color schemes. it'd be nice to have a backwards compatible way of supporting this.

maybe in `fish_prompt` and `fish_mode_prompt` where we check `type -q bobthefish_colors`, add a block and after calling their custom colors, check for `$color_nvm` and override `$color_node`?

Reply:
I'm not sure how to do this in `fish_prompt` or `fish_mode_prompt`, but I can figure it out within `__bobthefish_prompt_node`. I'll push that up. If you'd rather go another route, I'll need your help.

Reply:
Actually, yeah, it's probably fine to do it here. It's not like anything else needs the node color :)

Reply:
Never mind. I figured it out.

Reply:
> Actually, yeah, it's probably fine to do it here. It's not like anything else needs the node color :)

Hmm, good point, I'll opt for that instead. Then it's only one place.